### ISSUE TITLE
Springboard crash on odysseyra1n(?)

### ISSUE THREAD

**Issue by sevenpastzeero** [himynameisubik/Kalm-support#153]
General information:
===

**Kalm version:** 2.0.2

**Kalm preferences file:**
https://pastebin.com/jRSxJUeP

Device information:
===

**iOS version and language:** 13.6 beta

**Device:** iPhone 7 plus

**Jailbreak:** Odysseyra1n

Bug description:
===

**Describe the bug:** Lockscreen crashed while browsing notifications.
I switched to odysseyra1n yesterday, and everything looks fine. But today, while the device was locked, I woke it up to browse incoming notifications, and springboard crashed. Cr4ashed log says it is kalm.

**How to reproduce:**

**Expected behavior:**

**Screenshots:**

Additional information:
===

**Tweak list:**
https://pastebin.com/Ka8kteyb

**Crashlog:**
https://pastebin.com/RqYgHTUw

**Comment by himynameisubik** [comment]
I doubt this is linked to Odysseyra1n. It works fine on my test device with it. Did you try these steps first? https://github.com/himynameisubik/Kalm-support#general-steps-to-try-before-creating-an-issue

**Comment by sevenpastzeero** [comment]
I tried the steps mentioned in the link. Tbh, it happened only once in the past two days since I switched to odysseyra1n.
I couldn't reproduce the issue again. What happened before the crash, was that I had 4 different notifications with 2 of them grouped. I collapsed the first group to read the messages, the scrolled up and collapsed the second one, while scrolling through the notifications, sb crashed.
The reason I said it might be odysseyra1n, is because this never happened on checkra1n since I bought the tweak two weeks ago.
